packages/terra-date-picker/docs/README.md

```diff
@@ -1,8 +1,18 @@
 # Terra Date Picker
 
-DatePicker is a controlled input component that provides users a way to enter or select a date from the date picker. DatePicker is essentially a wrapper for [react-datepicker][1] and leverages many of its props. One important difference between DatePicker and [react-datepicker][1] is that all of the date props in [react-datepicker][1] must be a [moment][2] object whereas the date props in DatePicker are ISO 8601 representation of the date.
+DatePicker is an input component that provides users a way to enter or select a date from the date picker. DatePicker is essentially a wrapper for [react-datepicker][1] and leverages many of its props. One important difference between DatePicker and [react-datepicker][1] is that all of the date props in [react-datepicker][1] must be a [moment][2] object whereas the date props in DatePicker are ISO 8601 representation of the date.
```

Comment:
Not related to these changes, but we should probably remove the references to react-datepicker since we pulled that code into terra-date-picker

Reply:
Updated. https://github.com/cerner/terra-framework/pull/738/commits/216700ce9829692129df8c0b3853af7a24c2e4f5